x-pack/test/functional/apps/security/field_level_security.js

```diff
@@ -125,8 +125,9 @@ export default function ({ getService, getPageObjects }) {
         expect(hitCount).to.be('2');
       });
       const rowData = await PageObjects.discover.getDocTableIndex(1);
+      // runtime_customer_ssn is visible, but it has no value
       expect(rowData).to.be(
-        'customer_name:ABC Company customer_region:WEST _id:2 _type: - _index:flstest _score:0'
+        '_id:2 _type: - _index:flstest _score:0 customer_name.keyword:ABC Company customer_region.keyword:WEST runtime_customer_ssn: customer_region:WEST customer_name:ABC Company'
```

Comment:
This is a tricky one and I'm not sure what the expected behavior here is.  My understanding is that all fields will be present in the “fields” part of the response (need to verify this) but looks like they won’t have any value. From the permission perspective, is this ok? Is exposing the field in Discover like that ok?